vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/data/dif/PresentationSubmissionValidator.kt

```diff
@@ -0,0 +1,89 @@
+package at.asitplus.wallet.lib.data.dif
+
+import at.asitplus.KmmResult
+import kotlinx.serialization.Serializable
+
+@Serializable
+sealed interface PresentationSubmissionValidator {
+    companion object {
+        @Throws(MissingInputDescriptorGroupException::class)
```

Comment:
If it retruns a `KmmResult`, why does it also throw?

Reply:
fixed in commit https://github.com/a-sit-plus/kmm-vc-library/commit/65b06496293f5ec3ea9babe119596f8a137085d8